package.json

```diff
@@ -25,7 +25,7 @@
         "husky": "^7.0.4",
         "jest": "^27.4.5",
         "moment": "^2.29.1",
-        "obsidian": "^0.13.11",
+        "obsidian": "^0.15.4",
```

Comment:
The most recent update was obsidian 0.15.6 that caused the breaking change. Am I misunderstanding with this version being at 0.15.4?

Reply:
Perhaps. 15.4 was an insider release which changed `getActiveLeaf`. See release notes: https://discord.com/channels/686053708261228577/716028884885307432/993913278638084216

Reply:
gotcha, that makes sense! thanks